modules/util/jasper-jspc/build.gradle

```diff
@@ -2,13 +2,14 @@ sourceCompatibility = "1.6"
 targetCompatibility = "1.6"
 
 dependencies {
-	compile(group: "com.liferay", name: "com.liferay.portal.servlet.jsp.compiler", version: "1.0.5") {
-		exclude group: "com.liferay.portal"
-		exclude group: "javax.servlet"
-		exclude group: "javax.servlet.jsp", module: "jsp-api"
-	}
-
+	compile group: "com.liferay", name: "com.liferay.portal.osgi.web.servlet.jsp.compiler", version: "5.0.0"
```

Comment:
Hi @dantewang @yuhai ,
JSPC is just a tool in util folder, it should have its own dependencies if needed. We should not modify the production code for a tool.

Tina

Reply:
@tinatian Got it.